examples/v0/account_management/create_customer.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example illustrates how to create a new customer under a given
+manager account.
+
+Note by default this account will only be accessible via its parent AdWords
+manager account.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id):
+    customer_service = client.get_service('CustomerService')
+    resource_name = customer_service.customer_path(customer_id)
+    customer = client.get_type('Customer')
+    customer.resource_name = resource_name
+    customer.descriptive_name.value = "A new customer."
+    customer.currency_code.value = "USD"
+    customer.time_zone.value = "America/New_York"
+    customer.tracking_url_template.value = "{lpurl}?device={device}"
+    customer.final_url_suffix.value = "param1=value1&param2=value2"
+    customer.has_partners_badge.value = False
+
+    try:
+      response = customer_service.create_customer_client(
+        customer_id, customer)
```

Comment:
4 spaces indentation.

Reply:
Good catch, fixed this so it's indented by 4 not 2.